packages/shared/src/messages/MessageProxy.ts

```diff
@@ -174,25 +167,33 @@ export class RpcBrowser {
     });
   }
 
-  getProxy<T>(classType: { CHANNEL: string; prototype: T }): T {
-    // eslint-disable-next-line @typescript-eslint/no-this-alias
-    const thisRef = this;
+  getProxy<T>(channel: RpcChannel<T>, options?: { noTimeoutMethods: Array<keyof T> }): T {
```

Comment:
if we says T must extend ObjectInstance, does it automatically infers keyof T as string?